install.sh

```diff
@@ -301,7 +301,12 @@ echo_docker_as_nonroot() {
 	echo "         documentation for details: https://docs.docker.com/go/attack-surface/"
 	echo
 	echo "================================================================================"
-	echo
+	echo 
+	echo "Hint: You might want to grant Docker access to the current user by executing:"
+	echo 
+	echo '         sudo usermod -aG docker $USER'
+	echo 
+	echo "================================================================================"
```

Comment:
This was removed on purpose in favor of `dockerd-rootless-setuptool.sh install`:
- https://github.com/docker/docker-install/pull/215

Reply:
Correct; links were added for rootless, and the https://docs.docker.com/go/daemon-access/ link provides information how to run as a non-root user.

We prever using those links, as both provides relevant information on making these changes, and also outline the risks of doing so (having access to the API allows a non-root user to escalate to `root` on the host; https://docs.docker.com/go/attack-surface/).